src/setUnion.js

```diff
@@ -0,0 +1,25 @@
+export default function setUnion(arr1,arr2){
+
+let results = []
+const normalizeLength = Math.max(arr1.length, arr2.length)
```

Comment:
Change this variable name to `normalizedLength` to make it clear that it's not a function that is normalizing length (verb), but rather the normalized length result.